frontend_tests/node_tests/stream_data.js

```diff
@@ -635,3 +641,38 @@ zrequire('marked', 'third/marked/lib/marked');
 
     assert.equal(page_params.notifications_stream, "foo");
 }());
+
+(function test_invite_streams() {
+    // add default stream
+    var merry = {
+        email: 'merry@chrismas.com',
+        user_id: 25,
+        full_name: 'Me Myself',
+        timezone: 'US/Pacific',
+    };
+
+    var inviter = {
+        stream_id: 25,
+        name: 'Inviter',
+        subscribed: true,
+    };
+
+    var orie = {
+        stream_id: 320,
+        name: 'Orie',
+        subscribed: true,
+    };
+
+    // clear all the data form stream_data, and people
+    stream_data.clear_subscriptions();
+    people.init();
+
+    stream_data.add_sub('Orie', orie);
+    stream_data.add_sub('Inviter', inviter);
+    stream_data.add_subscriber('Inviter', merry.user_id);
+    stream_data.set_realm_default_streams([orie]);
+    people.add(merry);
```

Comment:
Do we need "merry" for this test to pass?  If not, it seems like a distracting detail.  Also, in English, "merry" is generally not a person's name, whereas "mary@example.com" would be.

Reply:
Addressed both nits.